index.js

```diff
@@ -1,4 +1,11 @@
 const honkify = () => {
+
+  //check if the user is on iOS Safari, if they are, we initiate the safari audio "hack"
+  var ua = window.navigator.userAgent;
+  var iOS = !!ua.match(/iPad/i) || !!ua.match(/iPhone/i);
+  var webkit = !!ua.match(/WebKit/i);
+  var iOSSafari = iOS && webkit && !ua.match(/CriOS/i);
```

Comment:
Can we move this after the `window` check so we don’t hit SSR issues?

Also — I know I shouldn’t care, but can you swap these to `const` instead?

Reply:
Just pushed the changes. 

Sorry for the lack of ES6 lol, wrote this in a hurry in English class